src/main/java/thePackmaster/relics/bladestormpack/FlyingDutchman.java

```diff
@@ -0,0 +1,85 @@
+package thePackmaster.relics.bladestormpack;
+
+import com.megacrit.cardcrawl.actions.common.RelicAboveCreatureAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.packs.BladeStormPack;
+import thePackmaster.powers.bladestormpack.WindrushPower;
+import thePackmaster.powers.instadeathpack.Precision;
+import thePackmaster.relics.AbstractPackmasterRelic;
+import thePackmaster.util.Wiz;
+
+import static com.megacrit.cardcrawl.dungeons.AbstractDungeon.player;
+
+//REFS: Necronomicon & Shuriken & Centennial Puzzle (base game)
+public class FlyingDutchman extends AbstractPackmasterRelic {
+    public static final String ID = SpireAnniversary5Mod.makeID("FlyingDutchman");
+    public static final int WINDRUSH = 2;
+    public static final int PRECISION = 3;
+    public static final int ATTACKS_FOR_TRIGGER = 1;
+
+    public FlyingDutchman() {
+        super(ID, RelicTier.COMMON, LandingSound.FLAT, BladeStormPack.ID);
+        this.counter = 0;
+    }
+
+    @Override
+    public void onEquip() {
+        CardCrawlGame.sound.play("NECRONOMICON");
+        init();
+    }
+
+    @Override
+    public void atTurnStart() {
+        init();
+    }
+
+    public void init() {
+        grayscale = false;
+        counter = 0;
+        pulse = true;
+    }
+
+    @Override
+    public void onPlayCard(AbstractCard c, AbstractMonster m) {
+        if (grayscale || c.type != AbstractCard.CardType.ATTACK) { return; }
+
+        if (counter >= ATTACKS_FOR_TRIGGER) {
+            Wiz.applyToSelf(new WindrushPower(player, WINDRUSH));
+            Wiz.applyToSelf(new Precision(player, PRECISION));
+
+            pulse = false;
+            addToTop(new RelicAboveCreatureAction( player, this));
+            flash();
+
+            this.grayscale = true;
```

Comment:
I'm pretty sure the base game only uses `grayscale` for relics that no longer do anything for the rest of combat. Think Fossilized Helix or Horn Cleat.

Instead of using grayscale for this, you can take the same approach as Necronomicon and have a `private boolean activated;` field. Basically just make that field and switch out all the uses of grayscale to use that field instead, and everything should work fine.